.github/workflows/nightlies.yml

```diff
@@ -54,11 +54,12 @@ jobs:
         run: |
           python -m pip install --upgrade pip setuptools wheel
           # c.f. https://github.com/Anaconda-Platform/anaconda-client/issues/540
-          python -m pip install git+https://github.com/Anaconda-Server/anaconda-client
+          python -m pip install git+https://github.com/Anaconda-Server/anaconda-client@be1e14936a8e947da94d026c990715f0596d7043
```

Comment:
While this does run the risk of things breaking if this commit were to ever be removed from history, pinning it to something seems much safer then having an API break happen with no warning.